README.md

```diff
@@ -93,6 +93,11 @@ location.
 # Installation
 
 ## Installing with pip
+Check out the Mac setup guide for vrs-python [here](https://github.com/ga4gh/vrs-python/blob/main/docs/setup_help/m1_mac_setup.md). This is a crucial step before diving into UTA. Once you've completed the setup, proceed with configuring the PostgreSQL database for the Unified Transcriptome Annotation (UTA) application  [here](https://github.com/ga4gh/vrs-python/blob/main/docs/setup_help/uta_installation.md).   
```

Comment:
I'm sorry. I messed up here. I forgot that VRS-Python uses docker containers to run. We should note that this is only for local install of UTA, which stands for [Universal Transcript Archive](https://github.com/biocommons/uta)